client/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Schmoogle homepage</title>
+    <link rel="stylesheet" href="/client/stylesheet.css">
+    <script defer src="/client/DOM.js"></script>
+</head>
+<body>
+    <header>
+        <nav>
```

Comment:
nav is good but could have been better if you added icons inported from google fonts.
            <span class="material-icons header_apps"> apps </span>
            <span class="material-icons"> account_circle </span>